calabash/Classes/FranklyServer/Routes/LPConditionRoute.m

```diff
@@ -199,7 +204,9 @@ - (BOOL) atLeastOneAnimatingOnMainThreadWithQuery:(id) query {
                                                       NSUInteger idx,
                                                       BOOL *stop) {
             CAAnimation *animation = [view.layer animationForKey:key];
-            if (animation.duration > kLPConditionRouteAnimationDurationLimit) {
+            if ([key  isEqual: @"UITextSelectionViewCaretBlinkAnimation"]) {
```

Comment:
Same thing here: Extra space between `key` and `isEqual`
```suggestion
            if ([key isEqual: @"UITextSelectionViewCaretBlinkAnimation"]) {
```

Reply:
if ([key  isEqual: @"UITextSelectionViewCaretBlinkAnimation"]) {

Is that more of a thing regarding documentation?

Looks like we have three conditions:
1.  if ([key  isEqual: @"UITextSelectionViewCaretBlinkAnimation"]) {
2. else if (animation.duration > kLPConditionRouteAnimationDurationLimit) && not 1.
3. (basically) else

I like @jamuc's idea, but in this case it might be fine to combine both statements without having a dedicated method:
```suggestion
            if (animation.duration > kLPConditionRouteAnimationDurationLimit &&
               ! [key  isEqual: @"UITextSelectionViewCaretBlinkAnimation"]) { 
```
since the code itself seems legible enough.